editors/code/src/extension.ts

```diff
@@ -152,15 +152,15 @@ export function activate(context: vscode.ExtensionContext) {
     if (Server.config.displayInlayHints) {
         const hintsUpdater = new HintsUpdater();
         hintsUpdater.loadHints(vscode.window.activeTextEditor).then(() => {
-            vscode.window.onDidChangeActiveTextEditor(editor =>
+            disposeOnDeactivation(vscode.window.onDidChangeActiveTextEditor(editor =>
                 hintsUpdater.loadHints(editor)
-            );
-            vscode.workspace.onDidChangeTextDocument(e =>
+            ));
+            disposeOnDeactivation(vscode.workspace.onDidChangeTextDocument(e =>
                 hintsUpdater.updateHints(e)
-            );
-            vscode.workspace.onDidChangeConfiguration(_ =>
+            ));
+            disposeOnDeactivation(vscode.workspace.onDidChangeConfiguration(_ =>
                 hintsUpdater.toggleHintsDisplay(Server.config.displayInlayHints)
-            );
+            ));
```

Comment:
```rust
-            disposeOnDeactivation(vscode.window.onDidChangeActiveTextEditor(editor =>
-                hintsUpdater.loadHints(editor)
-            ));
-            disposeOnDeactivation(vscode.workspace.onDidChangeTextDocument(e =>
-                hintsUpdater.updateHints(e)
-            ));
-            disposeOnDeactivation(vscode.workspace.onDidChangeConfiguration(_ =>
-                hintsUpdater.toggleHintsDisplay(Server.config.displayInlayHints)
-            ));
+            disposeOnDeactivation(
+                vscode.window.onDidChangeActiveTextEditor(editor =>
+                    hintsUpdater.loadHints(editor)
+                )
+            );
+            disposeOnDeactivation(
+                vscode.workspace.onDidChangeTextDocument(e =>
+                    hintsUpdater.updateHints(e)
+                )
+            );
+            disposeOnDeactivation(
+                vscode.workspace.onDidChangeConfiguration(_ =>
+                    hintsUpdater.toggleHintsDisplay(
+                        Server.config.displayInlayHints
+                    )
+                )
+            );
```

Reply:
Actually this could probably be `vscode.window.onDidChangeActiveTextEditor(hintsUpdater.loadHints)`.

Reply:
Fixed.

Reply:
CI seems happy now.